web/apps/benchmarks-and-reports/next.config.js

```diff
@@ -7,15 +7,19 @@ let config = {
   eslint: {
     ignoreDuringBuilds: true,
   },
-  reactStrictMode: true,
   transpilePackages: ["@risc0/ui"],
   experimental: {
     caseSensitiveRoutes: true,
+    reactCompiler: false,
```

Comment:
need to turn off until https://github.com/TanStack/table/issues/5567#issuecomment-2131242574 is fixed